web/ui/src/Health/serviceHealth.js

```diff
@@ -56,10 +56,10 @@
                 service.instances.forEach(instance => {
 
                     // create a new status rollup for this instance
-                    instanceUniqueId = serviceId +"."+ instance.id;
+                    instanceUniqueId = serviceId +"."+ instance.model.InstanceID;
```

Comment:
instance id is now a very very long id string, so using `model.InstanceID` which is the instance number (like 0, or 1, or 2, or 3, or 4, or 5)